Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM node:12.10.0-alpine
+FROM node:14
```

Comment:
Couldn't use alpine distributon anymore because `aapt` tool I wanted to use to extract app version requires glibc which is not available on Alpine.
I thought droping Alpine is more reasonable than installing some compat library